pkg/agent/controller/agent.go

```diff
@@ -288,10 +402,59 @@ func (a *Controller) newServiceImport(svcExport *lighthousev2a1.ServiceExport) *
 				originName:      svcExport.Name,
 				originNamespace: svcExport.Namespace,
 			},
+			Labels: map[string]string{
+				labelSourceName:      svcExport.Name,
+				labelSourceNamespace: svcExport.Namespace,
+				labelSourceCluster:   a.clusterID,
+			},
 		},
 	}
 }
 
+func (a *Controller) getServiceImport(svcExport *lighthousev2a1.ServiceExport) (*lighthousev2a1.ServiceImport, error) {
+	siName := svcExport.Name + "-" + svcExport.Namespace + "-" + a.clusterID
```

Comment:
This is duplicated with L400 - please put in a `serviceImportNameFrom` function.

Reply:
Done